extra_docs/sensitivities/covariance_sensitivity.tex

```diff
@@ -0,0 +1,255 @@
+$$ \mu = 
+\begin{bmatrix}
+\mu_1 \\
+\mu_2 \\
+\vdots \\
+\mu_m
+\end{bmatrix}
+= 
+\begin{bmatrix}
+\frac{1}{n} \sum_{i=1}^n x_{1i} \\[6pt]
+\frac{1}{n} \sum_{i=1}^n x_{2i} \\[6pt]
+\vdots\\
+\frac{1}{n} \sum_{i=1}^n x_{mi} \\
+\end{bmatrix}.
+$$
+Then, the covariance matrix of $X$ may be written as 
+$$\frac{1}{n-1}\sum_{i=1}^n (X_i - \mu)(X_i-\mu)^T.$$
+\end{lemma}
+
+\begin{proof}
+\begin{align*}
+\frac{1}{n-1} \sum_{i=1}^n (X_i - \mu)&(X_i-\mu)^T = \frac{1}{n-1} \sum_{i=1}^n
+\begin{bmatrix}
+x_{1i} - \mu_1 \\
+x_{2i} - \mu_2 \\
+\vdots\\
+x_{mi} - \mu_m
+\end{bmatrix} 
+\begin{bmatrix}
+x_{1i} - \mu_1 & 
+x_{2i} - \mu_2 &
+\cdots &
+x_{mi} - \mu_m
+\end{bmatrix}\\
+&= \frac{1}{n-1} \sum_{i=1}^n 
+\begin{bmatrix}
+(x_{1i}-\mu_1)(x_{1i}-\mu_1) & (x_{1i}-\mu_1)(x_{2i}-\mu_2) & \cdots & (x_{1i}-\mu_1)(x_{mi}-\mu_m)\\
+(x_{2i}-\mu_1)(x_{1i}-\mu_1) & (x_{2i}-\mu_1)(x_{2i}-\mu_2) & \cdots & (x_{2i}-\mu_1)(x_{mi}-\mu_m)\\
+\vdots & \vdots & \ddots & \vdots \\
+(x_{mi}-\mu_1)(x_{1i}-\mu_1) & (x_{mi}-\mu_1)(x_{2i}-\mu_2) & \cdots & (x_{mi}-\mu_1)(x_{mi}-\mu_m)\\
+\end{bmatrix}\\
+\end{align*}
+$$
+=
+\begin{bmatrix}
+ \frac{1}{n-1} \sum_{i=1}^n (x_{1i}-\mu_1)(x_{1i}-\mu_1) &  \frac{1}{n-1} \sum_{i=1}^n (x_{1i}-\mu_1)(x_{2i}-\mu_2) & \cdots &  \frac{1}{n-1} \sum_{i=1}^n (x_{1i}-\mu_1)(x_{mi}-\mu_m)\\[6pt]
+ \frac{1}{n-1} \sum_{i=1}^n (x_{2i}-\mu_1)(x_{1i}-\mu_1) &  \frac{1}{n-1} \sum_{i=1}^n (x_{2i}-\mu_1)(x_{2i}-\mu_2) & \cdots &  \frac{1}{n-1} \sum_{i=1}^n (x_{2i}-\mu_1)(x_{mi}-\mu_m)\\[6pt]
+\vdots & \vdots & \ddots & \vdots \\
+ \frac{1}{n-1} \sum_{i=1}^n (x_{mi}-\mu_1)(x_{1i}-\mu_1) &  \frac{1}{n-1} \sum_{i=1}^n (x_{mi}-\mu_1)(x_{2i}-\mu_2) & \cdots &  \frac{1}{n-1} \sum_{i=1}^n (x_{mi}-\mu_1)(x_{mi}-\mu_m)\\[6pt]
+\end{bmatrix}
+$$
+which is the covariance matrix of $X$.
+\end{proof}
+
+\begin{lemma}
+\label{lemma:sum0}
+$$\sum_{i=1}^n (X_i - \mu) = 0.$$
+\end{lemma}
+
+\begin{proof}
+\begin{align*}
+\sum(X_i - \mu) &= \sum_{i=1}^n \left(
+\begin{bmatrix}
+x_{1i}\\
+x_{2i}\\
+\vdots \\
+x_{mi}
+\end{bmatrix}
+-
+\begin{bmatrix}
+\mu_1\\
+\mu_2\\
+\vdots \\
+\mu_m
+\end{bmatrix}
+\right)\\
+&= \sum_{i=1}^n
+\begin{bmatrix}
+x_{1i} - \mu_1 \\
+x_{2i} - \mu_2 \\
+\vdots\\
+x_{mi} - \mu_m
+\end{bmatrix}\\
+&= 
+\begin{bmatrix}
+\sum_{i=1}^n (x_{1i} - \mu_1) \\
+\sum_{i=1}^n (x_{2i} - \mu_2) \\
+\vdots\\
+\sum_{i=1}^n (x_{mi} - \mu_m)
+\end{bmatrix} \\
+&= 
+\begin{bmatrix}
+n\mu_1 - n\mu_1\\
+n\mu_2 - n\mu_2 \\
+\vdots\\
+n\mu_m - n\mu_m \\
+\end{bmatrix}\\
+&= 0
+\end{align*}
+\end{proof}
+
+\begin{corollary}
+\label{corollary:sum0}
+$$\sum_{i=1}^n (X_i - \mu)^T = 0.$$ by identical proof construction as Lemma 1.
+\end{corollary}
+
+\begin{theorem}
+Let $F(X)$ be the covariance matrix of $X$ without the normalization factor of $n-1$. Let $M_i$ be a maximum bound on $x_i \in X_i$, and let $m_i$ be a minimum bound on $x_i \in X_i$. Then each entry $f_{ij}$ of this matrix has sensitivity bounded above by
```

Comment:
I have a related question to what I said on the variance sensitivity doc. What if X' has one fewer (rather than one more) observation than X? This might be covered later when we `consider the sensitivity of X'`, I just don't fully understand the framing of this as sensitivity of a random variable rather than of a function.

Reply:
I updated docs to say sensitivity of f(X').